src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy

```diff
@@ -722,15 +732,18 @@ class K8sServiceImpl implements K8sService {
     void deleteJob(String name) {
         k8sClient
                 .batchV1Api()
-                .deleteNamespacedJob(name, namespace, null, null, null, null,"Foreground", null)
+                .deleteNamespacedJob(name, namespace)
+                .execute()
```

Comment:
`"Foreground"` is not specified anymore?

Reply:
added now